ready-to-use fields/SEO/export.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "name": "Share / SEO Description",
+    "handle": "seoSummary",
+    "instructions": null,
+    "required": null,
+    "searchable": true,
+    "translationMethod": "none",
+    "translationKeyFormat": null,
+    "type": "craft\\fields\\PlainText",
+    "settings": {
+      "uiMode": "normal",
+      "placeholder": null,
+      "code": false,
+      "multiline": true,
+      "initialRows": 2,
+      "charLimit": 300,
+      "byteLimit": null,
+      "columnType": null
+    }
+  },
+  {
+    "name": "Share / SEO Title",
+    "handle": "seoTitle",
+    "instructions": null,
+    "required": null,
+    "searchable": true,
+    "translationMethod": "none",
+    "translationKeyFormat": null,
+    "type": "craft\\fields\\PlainText",
+    "settings": {
+      "uiMode": "normal",
+      "placeholder": null,
+      "code": false,
+      "multiline": false,
+      "initialRows": 4,
+      "charLimit": null,
+      "byteLimit": null,
+      "columnType": null
+    }
+  },
+  {
+    "name": "Share Image",
+    "handle": "seoImage",
+    "instructions": null,
+    "required": null,
+    "searchable": true,
+    "translationMethod": "site",
+    "translationKeyFormat": null,
+    "type": "craft\\fields\\Assets",
+    "settings": {
+      "restrictLocation": false,
+      "restrictedLocationSource": "volume:15f6c121-6024-4dca-837b-fae66c03f83c",
+      "restrictedLocationSubpath": null,
+      "allowSubfolders": false,
+      "restrictedDefaultUploadSubpath": null,
+      "defaultUploadLocationSource": "volume:15f6c121-6024-4dca-837b-fae66c03f83c",
```

Comment:
What happens during an import of this field when the same asset volume is not defined or does not exist in the project?

Reply:
The fields get imported. (I see the _No fields imported_ message always, not sure why.)

When you use the image field in any entry there it shows a message:
<img width="608" alt="Screenshot 2023-01-24 at 4 29 53 PM" src="https://user-images.githubusercontent.com/30666052/214276411-3f940c15-390c-4397-b6c7-4a08f4c3a5e7.png">